### PR TITLE
mitos

### PR DESCRIPTION
**Pull request by bernt-matthias**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/mitos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  home: http://mitos.bioinf.uni-leipzig.de
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE 
+  summary: MITOS
+  dev_url: https://gitlab.com/Bernt/MITOS 
+
+extra:
+  recipe-maintainers:
+    - Bernt
```
Comment (bgruening) [inline recipes/mitos/meta.yaml]: that seems to be incorrect :)

File: recipes/mitos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    - mitos.webserver
+  commands:
+    - runmitos.py -h
+
+about:
+  home: http://mitos.bioinf.uni-leipzig.de
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE 
+  summary: MITOS
```
Comment (bgruening) [inline recipes/mitos/meta.yaml]: A little bit more context please :)

File: recipes/mitos/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  sha256: de00eecb47f5a67c4a27104c87b4d6e26f1c9eddc86369864f7aed1b486c21c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - biopython ==1.73
```
Comment (bgruening) [inline recipes/mitos/meta.yaml]: You can remove everything here, but python and pip afaik.

File: recipes/mitos/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: de00eecb47f5a67c4a27104c87b4d6e26f1c9eddc86369864f7aed1b486c21c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
```
Comment (bgruening) [inline recipes/mitos/meta.yaml]: I think this needs to be `noarch: generic` as you have also non-python deps afaik.

File: recipes/mitos/meta.yaml (hunk 1 above)
Comment (bernt-matthias) [reply recipes/mitos/meta.yaml]: Just took the gitlab user name. Changed it to my full name.

File: recipes/mitos/meta.yaml (hunk 2 above)
Comment (bernt-matthias) [reply recipes/mitos/meta.yaml]: done

File: recipes/mitos/meta.yaml (hunk 3 above)
Comment (bernt-matthias) [reply recipes/mitos/meta.yaml]: lets try

File: recipes/mitos/meta.yaml (hunk 4 above)
Comment (bernt-matthias) [reply recipes/mitos/meta.yaml]: changed

**Pull request merged**